src/components/Datetime.tsx

```diff
@@ -0,0 +1,192 @@
+import { useState } from 'react';
+
+const Datetime = () => {
+  const [typeOfState, setTypeOfState] = useState<'success' | 'error'>('success');
+  const [typeOfSize, setTypeOfSize] = useState<'sm' | 'md' | 'lg'>('lg');
+  const [typeOfInput, setTypeOfInput] = useState<'datetime-local' | 'date' | 'time'>('datetime-local');
+  const [noMinWidth, setNoMinWidth] = useState(false); // State for noMinWidth
+	const [isDatetimeDisabled, setIsDatetimeDisabled] = useState(false);
+
+  // Get current date and format it
+  const today = new Date();
+  const year = today.getFullYear();
+  const month = String(today.getMonth() + 1).padStart(2, '0');
+  const day = String(today.getDate()).padStart(2, '0');
+  const formattedDate = `${year}-${month}-${day}`;
+  
+
+  const handleNewState = (event: Event) => {
+    const newState = (event.target as HTMLTdsRadioButtonElement).value as 'error' | 'success';
+    setTypeOfState(newState);
+  };
+
+  const handleType = (event: Event) => {
+    const newTypeOfInput: 'datetime-local' | 'date' | 'time' = (event.target as HTMLTdsRadioButtonElement).value as 'datetime-local' | 'date' | 'time';
+    setTypeOfInput(newTypeOfInput);
+  };
+
+  const handleSize = (event: Event) => {
+    const newSize: 'sm' | 'md' | 'lg' = (event.target as HTMLTdsRadioButtonElement).value as 'sm' | 'md' | 'lg';
```

Comment:
```suggestion
    const newSize = (event.target as HTMLTdsRadioButtonElement).value as 'sm' | 'md' | 'lg';
```

Same here as you did above.